### DISCUSSION TITLE
Request a more logical page layout

### DISCUSSION THREAD

**Discussion by qiqingli** [Kong/insomnia#3671] [Ideas]
The upper part of the current layout basically has no content to show, it is too wasteful of space, I hope the lower part of the content can be moved up to make reasonable use of the page space.
![image](https://user-images.githubusercontent.com/20805098/116523079-6a1c3c80-a908-11eb-9927-ca0bfd221a5d.png)

**Comment by reynolek** [comment]
Hi @qiqingli -- We are actively redesigning this based on feedback from the community. It is still in its infancy, but know that your voice has been heard.
